duckdb/join-duckdb.R

```diff
@@ -5,13 +5,13 @@ cat("# join-duckdb.R\n")
 source("./_helpers/helpers.R")
 
 suppressPackageStartupMessages({
-  library("DBI", lib.loc="./duckdb/r-duckdb", warn.conflicts=FALSE)
-  library("duckdb", lib.loc="./duckdb/r-duckdb", warn.conflicts=FALSE)
+  library("DBI", lib.loc="./duckdb-latest/r-duckdb-latest", warn.conflicts=FALSE)
```

Comment:
there is a separate directory storing test scripts for duckdb-latest, any reason why duckdb (not latest) will be using latest repo?
unless it is not going into master branch...

Reply:
For this run we will have a duckdb 0.9.0 and duckdb 0.8.1-3 version in the report to show the improvement. Eventually we will show just one version

Reply:
sorry, responded to quickly. This should just be going to `duckdb`